index.html

```diff
@@ -156,7 +156,26 @@ <h3 class="frameworks"><span>Frameworks</span><img src="./images/arrow-right.svg
                   <h3 class="skills"><span>Skills</span><img src="./images/arrow-right.svg" alt=""></h3>
                 </div>
               </section>
-            
-  
+
+              <!--Contact info section-->
+              <section class="contact-info">
+                <article class="contact-headings"> 
+                  <h2>Contact me</h2>
+                   <p> If you have an application you are interested in developing, 
+                     a feature that you need built or a project that needs coding.
+                      I’d love to help with it  </p> 
+                  </article>
+
+                <form action="https://formspree.io/f/xgeddrdw" method="POST">
+
+                <input type="text" name="username"  maxlength="30" placeholder="Yeremias" required>
```

Comment:
please it would be better if you change the `placeholder` value like the one in the Figma design and put `Full name ` instead of your name